sandbox/apparmor/apparmor.go

```diff
@@ -731,6 +731,11 @@ func probeParserFeatures() ([]string, error) {
 			probe:   "mqueue,",
```

Comment:
wondering if it makes sense to keep this check knowing it's not as useful as we wanted it to be. Would it make sense to drop it in a followup?

Reply:
We use it in some super-power interfaces like steam IIRC.